internal/pkg/levee/levee.go

```diff
@@ -83,6 +82,16 @@ func run(pass *analysis.Pass) (interface{}, error) {
 	return nil, nil
 }
 
+func checkIfASourceCanReachASink(pass *analysis.Pass, sources []*source.Source, instr ssa.Instruction, propagations map[ssa.Node]propagation.Propagation) {
```

Comment:
I'm not sure about this name. `check` indicates that this is some kind of predicate, but that isn't the case.

`reportSourcesReachingSinks`, maybe?

Reply:
shouldn't this be reportSourcesReachingSink?

Reply:
I don't understand your question. :thinking: 
I am suggesting that you use `reportSourcesReachingSinks` instead of `checkIfASourceCanReachASink`.

Reply:
I am asking if its more accurate to to say reportSourcesReachingSink instead of reportSourcesReachingSink`s`?

Reply:
Oh. Thank you for bringing my attention to that `s`. Yes, without the `s` is more accurate.

Reply:
Done.